### ISSUE TITLE
Confusion regarding ordering of log and softmax in inference code

### ISSUE THREAD

**Issue by umangksinha** [MarvinTeichmann/ConvCRF#27]
Hi Marvin,

Great project! I am trying to utilize it in my own research and went through your convcrf.py code. I have a doubt around [this line](https://github.com/MarvinTeichmann/ConvCRF/blob/88053dfc3ba8c97c7e1292c833b31486d1a30ef3/convcrf/convcrf.py#L578) where if I am not wrong, if default_conf["logsoftmax"] is True you take the log followed by a softmax using the log_softmax function ([docs](https://pytorch.org/docs/master/nn.functional.html#torch.nn.functional.log_softmax)) but if it's false you take the softmax followed by the log. Can you please clarify on this?

Thanks
